pkg/provider/resource_cluster.go

```diff
@@ -22,7 +22,7 @@ func NewClusterResource() *ClusterResource {
 
 func (c *ClusterResource) Schema() *schema.Resource {
 	return &schema.Resource{
-		Description: "Create a Postgres Cluster",
+		Description: "The cluster resource is used to manage BigAnimal clusters. See https://www.enterprisedb.com/docs/biganimal/latest/getting_started/creating_a_cluster/ for more details.",
```

Comment:
```suggestion
		Description: "The cluster resource is used to manage BigAnimal clusters. See [Creating a cluster
 ](https://www.enterprisedb.com/docs/biganimal/latest/getting_started/creating_a_cluster/) for more details.",
```

Reply:
Clickable link is added at https://github.com/EnterpriseDB/terraform-provider-biganimal/commit/0aea7392df32d1d3b7fcb6e35524aa585fa8a660